bblfsh/pyuast.c

```diff
@@ -1,4 +1,4 @@
-#include "uast.h"
+#include "libuast/uast.h"
```

Comment:
I don't think this is correct, since `bblfsh` is not in the include list.

Reply:
Yeah, cruft from one of my tests, works because gcc looks also form the source code file path, but unneeded.